monitor.js

```diff
@@ -0,0 +1,117 @@
+const fetch = require('node-fetch')
```

Comment:
Can we use `import`? that will help to keep syntax consistent with other code

Reply:
I was refer to this file: https://github.com/ZenGPT/confluence-gpt/blob/main/routes/index.js

Reply:
I replaced to "import".